python/tests/visualization/test_draw.py

```diff
@@ -138,11 +138,11 @@ def hw_kernel():
     cudaq.set_target('ionq', emulate=True)
     # fmt: on
     expected_str = R"""
-     ╭───╮                                       ╭───╮╭─────╮╭───╮╭───╮
-q0 : ┤ h ├──●─────────────────────●──────────────┤ x ├┤ tdg ├┤ x ├┤ t ├
-     ╰───╯  │                     │              ╰─┬─╯╰─────╯╰─┬─╯├───┤
-q1 : ───────┼───────────●─────────┼───────────●────●───────────●──┤ t ├
-     ╭───╮╭─┴─╮╭─────╮╭─┴─╮╭───╮╭─┴─╮╭─────╮╭─┴─╮╭───╮ ╭───╮      ╰───╯
+     ╭───╮                                                        ╭───╮
+q0 : ┤ h ├──────────────●─────────────────────●────●───────────●──┤ t ├
+     ╰───╯              │                     │  ╭─┴─╮╭─────╮╭─┴─╮├───┤
+q1 : ───────●───────────┼─────────●───────────┼──┤ x ├┤ tdg ├┤ x ├┤ t ├
+     ╭───╮╭─┴─╮╭─────╮╭─┴─╮╭───╮╭─┴─╮╭─────╮╭─┴─╮├───┤╰┬───┬╯╰───╯╰───╯
```

Comment:
This seems odd. Why is this a different layout?

Reply:
This is merely because the order or control qubits changed (order or controls is irrelevant); the new implementation that again handles all controls by the same logic happens to preserve the order of controls as written in the source code, whereas the old implementation reversed it.